server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteRestoreSnapshotIT.java

```diff
@@ -579,6 +581,24 @@ public void testRestoreShallowSnapshotRepository() throws ExecutionException, In
         ensureGreen(restoredIndexName1);
         assertDocsPresentInIndex(client, restoredIndexName1, numDocsInIndex1);
 
+        // ensure recovery details are non-zero
+        RecoveryResponse recoveryResponse = client().admin().indices().prepareRecoveries(restoredIndexName1).execute().actionGet();
+        for (Map.Entry<String, List<RecoveryState>> entry : recoveryResponse.shardRecoveryStates().entrySet()) {
+            for (RecoveryState recoveryState : entry.getValue()) {
+                // ensure populated file details
+                assertThat(recoveryState.getIndex().totalFileCount(), greaterThan(0));
```

Comment:
Is it possible to add better tests for this? maybe we can replicate whatever we are doing for remote store restore ITs.